requirements.txt

```diff
@@ -2,3 +2,6 @@ flake8
 yamllint
 ome-model==6.0.0
 Pillow
+scikit-image
+https://github.com/manics/omero-upload/archive/pr1.zip
+https://github.com/manics/omero-rois/archive/master.zip
```

Comment:
I assume we will want https://github.com/ome/omero-rois deployed on PyPI first. Given https://github.com/ome/omero-rois/releases, do we want to go for `0.3.0` or `0.2.1` ?

Reply:
`0.3.0` since master is on `0.3.0.dev0`. Has the naming discussion been resolved?

Reply:
:+1: for the release.
Re naming if you are talking about `omero-rois` vs `omero-roi`, no-one commented on my suggestion so I do not think there is a resolution. We can certainly go ahead and get `omero-rois` published and rename later, the main downside will be the PyPI project might change and unlike GitHub, PyPI does not seamlessly redirect.